example-playbooks/manage_users/manage_users.yml

```diff
@@ -0,0 +1,80 @@
+#     but you can change the list of users and permissions by overriding the variable users.
+#     Every user defined in this playbook must be also defined in the inventory file under the section [cql_credentials] followed
+#     by its password.
+#     E.g.:
+#     [cql_credentials]
+#     scylla_admin=password1
+#     scylla_cql_monitor=password2
+#     customer_admin=password3
+#
+#     Operations allowed:
+#       * Addition of users
+#       * Addition of permissions for any user (Removing permissions isn't supported)
+#       * Changing password for any user except scylla_admin
+#
+#     Usage:
+#       ansible-playbook -i inventory.ini manage_users.yaml [-e "@users.yml"]
+
+- name: Add users from inventory and delete cassandra user
+  hosts: scylla
+  vars_files:
+    - vars/main.yml
+  tasks:
+    - name: Get broadcast_address and check if authentication is enabled
+      block:
+        - command: cat /etc/scylla/scylla.yaml
+          ignore_errors: true
+          register: _scylla_yaml_out
+
+        - set_fact:
+            _scylla_yaml_map: "{{ _scylla_yaml_out.stdout | from_yaml }}"
+
+        - set_fact:
+            broadcast_address: "{{ _scylla_yaml_map.broadcast_address }}"
+
+        - set_fact:
+            _authentication_enabled: |
+              {% if (_scylla_yaml_map.authenticator is defined and (_scylla_yaml_map.authenticator == 'TransitionalAuthenticator' or _scylla_yaml_map.authenticator == 'PasswordAuthenticator')) or
+              (_scylla_yaml_map.authorizer is defined and (_scylla_yaml_map.authorizer == 'TransitionalAuthorizer' or _scylla_yaml_map.authorizer == 'CassandraAuthorizer')) %}True{% else %}False{% endif %}
+
+    - fail:
+        msg: "Authentication is not enabled!"
+      when: _authentication_enabled|bool == false
+      run_once: true
+
+    - name: Set password for scylla_admin
+      set_fact:
+        scylla_admin_password: "{{ lookup('ini', scylla_admin_username, section='cql_credentials', allow_no_value=true, default=omit, file=inventory_file) }}"
+
+    - name: Create scylla_admin user
+      include_tasks: create_user.yml
+      vars:
+        superuser: true
+        admin_username: "cassandra"
+        admin_password: "cassandra"
+        username: "{{ scylla_admin_username }}"
+        password: "{{ scylla_admin_password }}"
+      ignore_errors: true
+      run_once: true
+
+    - name: Create other users
+      include_tasks: create_user.yml
+      vars:
+        superuser: "{{ item.value.superuser }}"
+        admin_username: "{{ scylla_admin_username }}"
+        admin_password: "{{ scylla_admin_password }}"
+        username: "{{ item.key }}"
+        password: "{{ lookup('ini', item.key, section='cql_credentials', allow_no_value=true, default=omit, file=inventory_file) }}"
+        permissions: "{{ item.value.permissions }}"
+      loop: "{{ users | dict2items }}"
+      run_once: true
+
+    - name: Delete cassandra user
```

Comment:
This task should be optional because maybe the user wants to keep `cassandra` username. Don't ask me why.

Reply:
Done